plugins/dashboards/frontend/public/templates/widgets/note/widget.html

```diff
@@ -7,8 +7,8 @@
     </div>
 	<div class="clyd-widget__content">
         <div style="width: 100%; height: 100%;" class="bu-is-flex bu-is-flex-direction-column bu-is-justify-content-center">
-            <div class="bu-pl-2 bu-pr-2" :style="widgetStyling">{{widgetText}}</div>
-            <p v-if="mylink" :style="linkStyling" class="bu-p-2"><a class="bu-pt-4 bu-is-clickable color-dark-blue-100" target="_blank" :href="mylink.link">{{mylink.text}}</a></p>
+            <div class="bu-pl-2 bu-pr-2" :style="widgetStyling" style="overflow: hidden">{{widgetText}}</div>
```

Comment:
I think it's better to use class instead of inline style. There's a helper class you can use here https://github.com/Countly/countly-server/blob/master/frontend/express/public/stylesheets/bulma/bulma-custom.css#L3246

Reply:
hmm, but wouldn't the hidden part just hide the rest of the text, I assume instead we want to break none breakable text and wrap it?

Reply:
https://github.com/Countly/countly-server/pull/3581/commits/d55722a1e1baf00d79a51c2ec8583f53b944620c
![image](https://user-images.githubusercontent.com/18065510/196675036-f250d51f-6b17-425d-90b6-aa12d6a2ef5e.png)
added class instead of inline style

Reply:
scrolling also added